modules/quality/CMakeLists.txt

```diff
@@ -1,2 +1,5 @@
 set(the_description "Image Quality Analysis API")
 ocv_define_module(quality opencv_core opencv_imgproc WRAP python)
+ocv_add_testdata(testdata/ contrib/quality
+    FILES_MATCHING PATTERN "*.dat"
```

Comment:
`testdata` should go into `opencv_extra` repository (create PR with the same source branch name).

We put into repository only small enough data files which is used/shared with "samples" (like text detection models).